pkg/controller/sparkapplication/sparkui.go

```diff
@@ -85,6 +88,12 @@ func createSparkUIIngress(app *v1beta2.SparkApplication, service SparkService, i
 			}},
 		},
 	}
+	if len(getIngressResourceAnnotations(app)) != 0 {
+		ingress.ObjectMeta.Annotations = getIngressResourceAnnotations(app)
+	}
+	if len(getIngressTlsHosts(app)) != 0 {
```

Comment:
Ditto, can you store the result of `getIngressTlsHosts ` in a variable?

Reply:
done